lib/SIL/SILPrinter.cpp

```diff
@@ -1176,6 +1176,78 @@ class SILPrinter : public SILInstructionVisitor<SILPrinter> {
     *this << ") : ";
     *this << BI->getType();
   }
+
+  // SWIFT_ENABLE_TENSORFLOW
+  void visitSymbolicValue(SymbolicValue v) {
+    switch (v.getKind()) {
+      case swift::SymbolicValue::Integer:
+        *this << v.getIntegerValue();
+        return;
+      case swift::SymbolicValue::Float: {
+        SmallString<12> decimal;
```

Comment:
Why is it 12? Is it an arbitrary number?

Reply:
`SmallString<12>` is what's used in [`visitFloatLiteralInst`](https://github.com/apple/swift/blob/9a6adcec9d09481091de9a10d8d45b4445a26995/lib/SIL/SILPrinter.cpp#L1210), I just copied that.
If you'd prefer a more auspicious number, let me know.